### ISSUE TITLE
MAUI 7.0.58 does not yet include this fix, however the upcoming 7.0.59 release will include it.

### ISSUE THREAD

**Issue by Vampeyer** [dotnet/maui#21899]
MAUI 7.0.58 does not yet include this fix, however the upcoming 7.0.59 release will include it.

We've made a couple of changes to mitigate the Windows OS bug:
1. We are setting `<PropertyGroup><WindowsAppSdkDeploymentManagerInitialize>false</WindowsAppSdkDeploymentManagerInitialize></PropertyGroup>` to disable the automatic code generation of the Deployment Manager by the Windows App SDK.
2. We are calling the Deployment Manager Initialize code ourselves and handling the result differently than the generated invocation from the Windows App SDK.

Ultimately the Windows OS level bug was causing the Deployment Manager's Initialize call to return a failed result (due to an Access Denied error, which is why running the app as Admin would fix this).  The generated code from the Windows App SDK would abrubtly invoke Environment.Exit on a failed result and did not log any useful information when this happened.  Previous versions of WindowsAppSDK (1.1.x) had a different order of evaluating MSBuild properties and did not automatically include the generated code to call this initialize method which is why we didn't see the bug in MAUI before this.

Since we are calling this Deployment Manager Initialize method ourselves now within MAUI instead, we are choosing to not exit immediately on a failed result, since in many cases the app will still work as expected.  This call is mostly important for apps using windows specific notification API's and a few others, which many MAUI apps will not be using as it installs the additional framework references that ship with the windows app runtime but can't be installed in all distribution scenarios.  It's correct to have this call in a MAUI app regardless of if you think you need it, the bug isn't in the fact that the deployment manager is being called, but rather in that on Windows 10 there was a bug where that call required elevation.

The reason this change works around the Windows 10 OS bug is that we are choosing to not fail if the deployment manager cannot complete because it's possible the app will still execute normally, which is the same behaviour MAUI apps had in previous releases.

To summarize, there are a few ways this issue can be mitigated:
1. Update to the latest [Windows 10 OS January 19](https://support.microsoft.com/en-us/topic/january-19-2023-kb5019275-os-builds-19042-2546-19044-2546-and-19045-2546-preview-8ae1b678-d38c-4249-848d-06b722e7c0ad) update
2. Manually install the runtime with the [Windows App SDK Runtime Installer](https://aka.ms/windowsappsdk/1.2/latest/windowsappruntimeinstall-x64.exe)
3. Set the project property in your MAUI app project: `<PropertyGroup><WindowsAppSdkDeploymentManagerInitialize>false</WindowsAppSdkDeploymentManagerInitialize></PropertyGroup>`
4. Update to MAUI 7.0.59 when it is released in the coming weeks

_Originally posted by @Redth in https://github.com/dotnet/maui/issues/12080#issuecomment-1398635826_

**Comment by Vampeyer** [comment]
I am on .NET 9.0 with the same issue.    
I am leery to develop on this tech stack now.

**Comment by drasticactions** [comment]
Closing as duplicate, https://github.com/dotnet/maui/issues/21901
